Opower integration support for multiple meters on single account (#30974)
home-assistant/home-assistant.github.io: source/_integrations/opower.markdown

Note the unit for gas is CCF (centum cubic feet). 1 CCF is one hundred cubic feet which is equivalent to 1 therm.

Depending on the utility and your setup, there may be multiple meters for a single utility account.

> If this is the case, your sensors will use a UUID to distinguish the data streams within that single account.

Reviewer: UUID is a rather technical term, can we avoid that?

Author: Removed it and replaced to explain that the id will not be tied back to anything the energy provider provides.

Note the unit for gas is CCF (centum cubic feet). 1 CCF is one hundred cubic feet which is equivalent to 1 therm.

> Depending on the utility and your setup, there may be multiple meters for a single utility account.
> If this is the case, your sensors will use a unique id that does not tie back to any identifier used by the utility to distinguish the data streams within that single account.

Reviewer: ```suggestion
Depending on the utility and your setup, multiple meters may exist for a single utility account.
If this is the case, your sensors will use a unique ID that does not tie back to any identifier used by the utility to distinguish the data streams within that single account.
```
